lib/ruby_indexer/lib/ruby_indexer/uri.rb

```diff
@@ -31,6 +31,7 @@ def from_path(path:, fragment: nil, scheme: "file", load_path_entry: nil)
         end
 
         uri = build(scheme: scheme, path: escaped_path, fragment: fragment)
+        uri.raw_path = path
```

Comment:
I dont think we should do that, we should rather split whole work into platform specifci path unscaping/escaping and then use the right code paths on the right platform, also the result can be cached on the entry level